### PR TITLE
Add Razor Network

### PR DESCRIPTION
**Pull request by yohanelly95**

### CODE REVIEW

File: _data/icons/razornetwork.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url": "ipfs://QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK",
+      "width": 1024,
+      "height": 1024,
```
Comment (ligi) [inline _data/icons/razornetwork.json:5]: size seems to be wrong
```
identify QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK         03:34:28
QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK PNG 900x900 900x900+0+0 8-bit sRGB 47186B 0.000u 0:00.000
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

File: _data/icons/razornetwork.json (hunk 1 above)
Comment (yohanelly95) [reply _data/icons/razornetwork.json]: This is resolved, I have updtedupdated the dimensions to `900x900`

**Comment by yohanelly95** [comment]
> Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json) e.g. run `npx prettier --write _data/*/*.json`

Resolved

**Comment by yohanelly95** [comment]
hey @ligi should I create a new PR for this ?

**Comment by yohanelly95** [comment]
@ligi I have reopened the PR here - https://github.com/ethereum-lists/chains/pull/2795

**Pull request closed without merging**